src/main/java/uk/gov/dvsa/mot/fixtures/mot/TesterDoesStepDefinitions.java

```diff
@@ -579,12 +580,12 @@ private void searchForVehicle(String registration, String vin, String siteName)
         if (registration != null && registration.trim().length() > 0) {
             //Check added because this is an area of previous defects - e.g. DVSA and DVLA vehicle ids being mixed up
             //And I check that the search results has Registration that was searched for
-            assertEquals(registration, driverWrapper.getTextFromTableColumn("Registration mark"));
+            assertTrue(driverWrapper.containsMessage(registration));
```

Comment:
Can we improve this - in the future, look for <div> and look for registration inside the <div>

Reply:
Done